docs/README-develop-resources-providers.md

```diff
@@ -471,62 +468,61 @@ $:.unshift(*Dir[File.expand_path('../../files/default/vendor/gems/**/lib', __FIL
 require 'cisco_node_utils'
 
 class Chef
-  class Provider::CiscoRouterEigrp < Provider
-    provides :cisco_router_eigrp
+  class Provider
+    # CiscoRouterEigrp provider for Chef
+    class CiscoRouterEigrp < Provider
+      provides :cisco_router_eigrp
+
+      def initialize(new_resource, run_context)
+        super(new_resource, run_context)
+        @router = nil
+        @name = new_resource.name
+      end
 
-    def initialize(new_resource, run_context)
-      super(new_resource, run_context)
-      @router = nil
-      @name = new_resource.name
-    end
+      def whyrun_supported?
+        true
+      end
 
-    def whyrun_supported?
-      true
-    end
+      # Find specific router instance with this name
+      def load_current_resource
+        @router = Cisco::RouterEigrp.routers[@name]
+      end
 
-    # Find specific router instance with this name
-    def load_current_resource
-      @router = Cisco::RouterEigrp.routers[@name]
-    end
+      def action_create
```

Comment:
Did you happen to test these changes using an eigrp recipe?  I only ask because teams next week are going to use this during their lab work.